ios/Info.plist.xml

```diff
@@ -52,5 +52,7 @@
         <string>LaunchScreen</string>
         <key>NSCameraUsageDescription</key>
         <string>Scan QR codes</string>
+        <key>NSFaceIDUsageDescription</key>
+        <string>Authentication required to access secrets</string>
```

Comment:
Please clarify in the message, what kind of authentication is required and what the user should do.

Reply:
The text is shown as a reason why the app wants to access Face ID. Do you have a suggestion what other apps show as a reason? My iOS devics don't have Face ID and what apps are showing as a reason for using Touch ID ranges from "<Apple ID>" (App Store connect from Apple itself) to "Use your finger" (my bank app)

Reply:
Changed